exports/scanresultscollector.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+import datetime
+import json
+import logging
+import os
+import random
+import string
+import tarfile
+import uuid
+from io import BytesIO
+from io import StringIO
+
+from google.cloud import storage
+
+
+class ScanResultsCollector:
+    """
+    Sends the result to a Google Cloud Storage bucket, for future use in DefectDojo
```

Comment:
"for future use" could be changed anytime, so this class should focus on exporting the results to the external storage only (while some data might need to be specific for a certain project)

Reply:
removed ref. to Defectdojo